app/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,9 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" /> <!-- Used by sentry. -->
 
+    <!--To fix lint error 'Hardware feature touchscreen not explicitly marked as optional'-->
+    <uses-feature android:name="android.hardware.touchscreen" android:required="true" />
```

Comment:
I think you don't need this commit: I checked out the commit above this, ran lint, and didn't get the error anymore. I think removing the leanback dependency fixed this failure.